client/assets/js/videoplayer.js

```diff
@@ -0,0 +1,22 @@
+angular.element(document).ready(function () {
+  var controls = {
+    video: angular.element("#myvideo")
```

Comment:
Rename #myvideo on something like #welcome-page-video, because it is not clearly